### PR TITLE
Adding networking doc

### PR DESCRIPTION
**Pull request by pdube**

### CODE REVIEW

File: source/includes/openstack/_networks.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+### Networks
+
+Networks are privisioned to be able to control data flow to instances. To comminucate with the external network, the network must be connected to a [router]#routers
```
Comment (prollynomial) [inline source/includes/openstack/_networks.md]: comminucate -> communicate

Comment (prollynomial) [inline source/includes/openstack/_networks.md]: Doesn't a link need brackets around the URL? (Talking about the `#routers` part.)

File: source/includes/openstack/_networks.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+<code>POST /services/<a href="#service-connections">:service_code</a>/<a href="#environments">:environment_name</a>/networks</code>
+
+Create a network in an environment.
+
+Optional | &nbsp;
+------ | -----------
+`name`<br/>*string* | Name of the network
+`cidr`<br/>*string* | The CIDR of the network. The mask should be between 1 and 30 inclusively. Example: 10.50.0.0/22
+`dhcpEnabled`<br/>*boolean* | If DHCP should be enabled for the network or if it will be handled by the user
```
Comment (prollynomial) [inline source/includes/openstack/_networks.md]: We should maybe mention that these will be auto-generated, and for the case of dhcpEnabled, what the default value is.

**Review by prollynomial — APPROVED** [review]
Couple of small changes, but looks good

**Review by simongodard — APPROVED** [review]

**Pull request merged**